R/see_wrappers.R

```diff
@@ -0,0 +1,36 @@
+# Wrappers for See Functions
+
+#' @rdname line_line_intersection_rcpp
+#' @param use Whether the R or C++ implementation is called.
+line.line.intersection = function(P1, P2, P3, P4, interior.only, use = 'cpp') {

```

Comment:
Note that the convention in R is to use '<-' for names, and '=' to arguments. The reason is that these two assignment operators have different meanings (please see: https://www.rdocumentation.org/packages/base/versions/3.6.2/topics/assignOps). In this instance, whereby you create a function, it doesn't matter (and I personally don't since I believe the goal of code is to ultimately work), but since I know you are really strict with 'code quality' I thought it would have been best to let you know?

Reply:
I know that it is common to use `<-` in R for assigning variables. The reason why I don't like it is that it is quite easy to make typos such as `a < -1`, which leads to a completely different result than `a <- 1`. These kind of typos can be very hard to find. I already used `=` in most of the previous code I wrote for the package, so it is perhaps a bit late to change this now. It doesn't seem to be a problem for CRAN or the tests in the GitHub actions, so I think this is fine.